### ISSUE TITLE
Web Interface, no way to create Category

### ISSUE THREAD

**Issue by BloodyIron** [nextcloud/notes#677]
I can find a way to create Notes Categories on the Android app, however when I'm using the web interface, in the Notes section, I cannot find a way to create a new Category. If the feature is there, it needs to be made more prominent.

However, I believe the feature is lacking in the web interface. So can we please have an obvious function to create a new Category and the mechanism on moving notes into/between Categories too please?

Categories are helpful, and I'd like to use them more. :)

**Comment by stefan-niedermann** [comment]
Top right, three dots menu, "Details", just enter the category you wish.

(I personal also find this bit hard, but yeah, this is the current state)

**Comment by BloodyIron** [comment]
Yeah, there's no way anyone will ever find that feature, let alone know it does that. I was intentionally trying to find it, and well I couldn't. This really needs to be moved to the left side near the Category section itself, and have a clear label "Create Category". Plus now that I think of it, I'm not seeing a way to delete a Category either (and naturally notes would need to move to Uncategorised in the same process, so you don't lose notes).

Form a UX perspective, this has a lot of room for improvement, can we have that added as a milestone somewhere or something please?

**Comment by stefan-niedermann** [comment]
Because one is not supposed to "create" and "delete" categories - you just **set** a category for a note.

What might look unfamiliar in the first place make sense if you think longer about it - why should you need to create a category instead of just setting it? Why should you clean up old empty categories? You just set categories for a note or remove it form a note. The app handles the purely technical aspects in the background.

> Form a UX perspective, this has a lot of room for improvement,

I agree, that the feature is not discoverable at the current position.

> can we have that added as a milestone somewhere or something please?

No, because we have to think about how it will be better *before* adding it to any milestone or start wild developments :laughing: 

cc @korelstar @nextcloud/designers

**Comment by BloodyIron** [comment]
Okay so I agree with the convenience of not even having to think about creating/deleting. But do Categories get actually Deleted when no Note is assigned to them?

Also, I agree about discussing the best way before assigning a milestone. I like where this is going, I was more just hoping this wouldn't fall to the wayside (as things do in many open source projects). But with continued dialogue, I hope this comes to fruition! Yay!

**Comment by korelstar** [comment]
Duplicate of #413

**Comment by korelstar** [comment]
> Okay so I agree with the convenience of not even having to think about creating/deleting. But do Categories get actually Deleted when no Note is assigned to them?

Yes

**Comment by korelstar** [comment]
#413 ist already in the milestone for the next non-patch version. Please see over there for details on how to improve this issue.
